kotlinx-coroutines-core/common/test/flow/operators/TimeoutTest.kt

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.flow.operators
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.*
+import kotlin.test.*
+import kotlin.time.Duration.Companion.milliseconds
+
+class TimeoutTest : TestBase() {
```

Comment:
Please add a test that uses `null` value and fails if you replace `send(it ?: NULL)` with `send(it)` in your implementation